shared/fs/index.stories.js

```diff
@@ -175,10 +174,6 @@ const provider = Sb.createPropProviderWithCommon({
     path,
     shouldShowReset: Types.pathToString(path).includes('reset'),
   }),
-  Usernames: (props: ConnectedUsernamesProps) => ({
```

Comment:
For some reason, we were mocking `Usernames` in the `fs` stories. Not sure why as we also mock it in `stories/prop-providers.js`

Reply:
Perhaps @songgao or @jzila knows?